### ISSUE TITLE
Pseudo elements are not reset

### ISSUE THREAD

**Issue by Dathix** [premasagar/cleanslate#35]
It seems to me that pseudo elements are not reset the way I expected it, maybe my expectations are just wrong or I did a mistake. 

For now, I just added this manually on top of my rules that uses important but maybe this should be included in cleanslate.

```
*::before, *::after {
   content: none;
}
```

**Comment by premasagar** [comment]
You are right, `::before` and `::after` should be added. The `content` property is the most important, since setting it to `none` will remove the pseudo-element. I wonder if all properties should be reset for pseudo-elements just as they are for elements?
